python/doc/examples/reliability_sensitivity/reliability/plot_crossentropy.py

```diff
@@ -0,0 +1,328 @@
+"""
+Cross Entropy Importance Sampling
+=================================
+"""
+# %%
+#
+# The objective is to evaluate a failure probability from the two versions of Cross Entropy Importance Sampling working in Standard or Physical spaces, see :class:`~openturns.StandardSpaceCrossEntropyImportanceSampling` and :class:`~openturns.PhysicalSpaceCrossEntropyImportanceSampling`.
```

Comment:
The doc is not so easy to read when the line is beyond the 90th column. Splitting the text into several lines will make the review much easier.

Reply:
Thanks. This has been corrected.